src/main/java/com/brandon3055/draconicevolution/handlers/CustomArmorHandler.java

```diff
@@ -118,7 +120,7 @@ public void onPlayerDeath(LivingDeathEvent event) {
             return;
         }
 
-        if (summery.protectionPoints > 500) {
+        if (summery.protectionPoints > 500 && player.getCapability(PlayerCapabilitiesProvider.PLAYER_CAP, null).getShieldState()) {
```

Comment:
Does onPlayerAttacked get called before a player dies?  I never actually tested if that was the case...  If so, then I can remove this check.

Reply:
Just tested this; it is needed when a player has over 500 shields, otherwise with shielding off, it makes the player invincible by resetting their HP to 10 with no penalty.

Reply:
Oh my bad. I wasn't paying close enough attention. I thought both checks were in the same event.